frame/vesting/src/lib.rs

```diff
@@ -244,17 +353,20 @@ pub mod pallet {
 		#[pallet::weight(T::WeightInfo::vest_other_locked(MaxLocksOf::<T>::get())
 			.max(T::WeightInfo::vest_other_unlocked(MaxLocksOf::<T>::get()))
 		)]
-		pub fn vest_other(origin: OriginFor<T>, target: <T::Lookup as StaticLookup>::Source) -> DispatchResult {
+		pub fn vest_other(
+			origin: OriginFor<T>,
+			target: <T::Lookup as StaticLookup>::Source,
+		) -> DispatchResult {
 			ensure_signed(origin)?;
-			Self::update_lock(T::Lookup::lookup(target)?)
+			let who = T::Lookup::lookup(target)?;
+			Self::do_vest(who)
 		}
 
 		/// Create a vested transfer.
 		///
 		/// The dispatch origin for this call must be _Signed_.
 		///
 		/// - `target`: The account that should be transferred the vested funds.
```

Comment:
This line is not really comprehensible to me. is it correct?

Reply:
Improved the wording